hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitForm.js

```diff
@@ -82,8 +82,6 @@ export const OrgUnitForm = ({
             const actualAliases = value.filter(alias => alias !== '');
             if (newAlias !== '' && !isEqual(actualAliases, orgUnitAliases)) {
                 setOrgUnitModified(true);
-            } else {
-                setOrgUnitModified(false);
```

Comment:
This is not fixing the issue in fact, this is avoiding it. Can you have a look into `OrgUnitInfosComponent` please. The `reformatOrgUnit` method looks weird. It's using `orgunit.parent` but we only have a `orgunit.parent_id`